wet-boew/geomap/_base.scss

```
@@ -0,0 +1,34 @@
+/* Geomap plugin GC override */
+.wb-geomap {
+	&.large-checkboxes {
+		.geomap-lgnd-layer.gc-chckbxrdio input[type="checkbox"] + label {
+			padding-left: 4px;
+
+			&::before {
+				left: 2px;
+			}
+
+			&::after {
+				left: 10px;
+			}
+		}
+
+		.wb-geomap-layers table tbody tr td .gc-chckbxrdio {
```

Do we need to be that specific?

Is this will work?


```suggestion
		.wb-geomap-layers .gc-chckbxrdio {
```